bubble-wrap-style.yaml

```diff
@@ -3876,6 +3085,29 @@ layers:
                         style: italic
                         size: 10px
                         stroke: { color: global.text_stroke_address, width: 4 }
+        entrance-labels:
+            filter:
+                $zoom: { min: 20 }
+                any:
+                    - kind: entrance
+                all:
+                    - function() { return global.text_visible_address; }
+            draw:
+                mapzen_icon_library:
+                    size: [[13, [95%,95%]], [16, [100%,100%]], [18, [105%,105%]]]
+                    sprite: generic #function() { return feature.kind; }
+                    sprite_default: generic
+                    interactive: global.sdk_interactive
+                    priority: 65 #function() { return (feature.min_zoom && Math.floor(feature.min_zoom * 1000)) || 65; }
+                    repeat_group: abc
```

Comment:
Are we intentionally using separate `repeat_group` logic here, or is this vestigial? Can't remember/tell.

Reply:
It's adding the entrance icons to the same repeat_group as the pois, because they are in different source layers. The only other explicate repeat_group we use is for shields. We set collision false for trees at some zooms.